tests/shape_poly_test.py

```diff
@@ -1097,6 +1097,10 @@ def f1(x, y):
     exp = jax.export.export(jax.jit(f1))(*args_specs)
     self.assertEqual(exp.in_avals[0], exp.in_avals[1])
 
+  def test_linalg_inv_bug_22489(self):
```

Comment:
Please move this test to be part of_POLY_SHAPE_TEST_HARNESSES, e.g., search for how "linspace" is tested. That way the test follows the same pattern as others and it is more complete (it actually runs the code).